docs/VnVPlan/VnVPlan.tex

```diff
@@ -787,6 +788,29 @@ \subsubsection{Work Orders}
 
 \end{enumerate}
 
+\subsubsection{Shop Lookup}
```

Comment:
Can you move this section to be after `Services` so that the order matches the SRS?